src/arch/src/aarch64/regs.rs

```diff
@@ -80,7 +80,7 @@ const NR_FP_VREGS: usize = 32;
 // The core register are represented by the user_pt_regs structure. Look for it in
 // arch/arm64/include/uapi/asm/ptrace.h.
 
-// This macro gets the offset of a structure (i.e `str`) member (i.e `field`) without having
+// This macro gets the offset of a member (`field`) within a structure (`container`)  without having
 // an instance of that structure.
 // It uses a null pointer to retrieve the offset to the field.
 // Inspired by C solution: `#define offsetof(str, f) ((size_t)(&((str *)0)->f))`.
```

Comment:
I think this comment needs a bit of rewording. I don't think we're using a null pointer any more.

Reply:
I've adjusted the comment, mostly removing the description and simply referencing bindgen.